src/main/java/spoon/support/reflect/declaration/CtElementImpl.java

```diff
@@ -536,4 +540,12 @@ public CtElement clone() {
 		rh.setValue(this, value);
 		return (E) this;
 	}
+
+	public CtPath getPath() {
+		try {
```

Comment:
why not removing the try/catch and let the exception go?

Reply:
The exception is supposed to be risen from CtElementPathBuilder().fromElement(CtElement el, CtElement from) when from is not a parent of el. So in this case, I think it can't happen and I didn't want to annoy the user with the handling of an exception which is not supposed to happen.
But still, we can let it go.

Reply:
"let it go" then :-)

Reply:
I have a counter proposal: Why not catch `CtPathException` (which should not happen in that context) and throw a `SpoonException` to show that it's an spoon implementation error. Further more it would avoid that the user has to handle it.